src/simperium/channel.js

```diff
@@ -775,7 +807,7 @@ LocalQueue.prototype.dequeueChangesFor = function( id ) {
 	var changes = [], sent = this.sent[id], queue = this.queues[id];
 
 	if ( sent ) {
-		delete this.sent[id];
+		this.sent[id];
```

Comment:
This change is what is [failing the test](https://travis-ci.org/Simperium/node-simperium/jobs/590249312). However it's a necessary change.

Reply:
did we mean to delete the whole line?